src/main/java/cinema/util/PropertiesUtil.java

```diff
@@ -0,0 +1,32 @@
+package cinema.util;
+
+import cinema.exception.DaoException;
+import lombok.experimental.UtilityClass;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+
+@UtilityClass
+public final class PropertiesUtil {
```

Comment:
why do you write final if you use @UtilityClass.

Reply:
delete final keyword